internal/ioutil/print.go

```diff
@@ -58,6 +59,20 @@ func PrintMetadataMap(w io.Writer, metadata map[string]string) error {
 	return tw.Flush()
 }
 
+func PrintCertMap(w io.Writer, certPaths []string) error {
+	tw := newTabWriter(w)
+	fmt.Fprintln(tw, "STORE TYPE\tSTORE NAME\tCERTIFICATE\t")
+	for _, cert := range certPaths {
+		fileName := filepath.Base(cert)
+		dir := filepath.Dir(cert)
+		namedStore := filepath.Base(dir)
+		dir = filepath.Dir(dir)
+		storeType := filepath.Base(dir)
```

Comment:
nit: we need to add a validation for storeType. Currently we support 3 store types: 'ca', 'signingAuthority' and 'tsa'.

The actual fix might need to go into `truststore.ListCerts` method.

Reply:
Good suggestion. But unlike adding a cert, for listing, we could just skip the invalid store types.

Reply:
Code updated.